test/gui/shared/scripts/helpers/api/Provisioning.py

```diff
@@ -51,3 +53,37 @@ def setup_app(app_name, action):
         disable_app(app_name)
     else:
         raise Exception("Unknown action: " + action)
+
+
+def create_group(group_name):
+    url = url_join(get_ocs_url(), 'groups')
+    body = {"groupid": group_name}
+    response = request.post(url, body)
+    if response.status_code != 200:
+        raise Exception(
+            "Creating group '%s' failed with %s\n" % (group_name, response.status_code)
+            + response.text
+        )
+    # save created group
+    created_groups[group_name] = True
```

Comment:
you might have to specify global var
```suggestion
    global created_groups
    created_groups[group_name] = True
```